addons/website_event/views/website_pages_views.xml

```diff
@@ -14,15 +14,25 @@
             <attribute name="action">open_website_url</attribute>
         </xpath>
 
+        <field name="address_id" position="attributes">
+            <attribute name="optional">hide</attribute>
+        </field>
+        <field name="date_end" position="attributes">
+            <attribute name="optional">hide</attribute>
+        </field>
+        <field name="seats_used" position="attributes">
```

Comment:
Sound quite important to me, isn't this quite critical to know for incoming events?
I guess it takes a lof of space to just display a number, but `Number of registrations` could probably have been renamed to something way smaller.
Or even better, what we were asking to the framework but "this is useless to everyone except website": an icon as column header with a tooltip, like a seat in this case :grimacing: 

Anyway, I guess this can be discussed later and shouldn't block anything

Reply:
@rdeodoo We still show the "Expected Attendees" which seems more useful as defined like this: `event.seats_unconfirmed + event.seats_reserved + event.seats_used` (so I guess `seats_used` is only an info about seats for which users have already paid or something like that). And if you judge it is "critical" to know: it's just hidden by default, it is still marked as optional and you can enable it (and it stays in your localstorage).

But I agree that list view labels could be renamed to something smaller and/or use an icon if the framework allowed it (we should still do it as a custo for our SEO/etc labels).